src/main/java/mods/eln/Eln.java

```diff
@@ -2692,12 +2752,12 @@ private void registerHeatFurnace(int id) {
             name = TR_NAME(Type.NONE, "Stone Heat Furnace");
 
             HeatFurnaceDescriptor desc = new HeatFurnaceDescriptor(name,
-                "stonefurnace", 1000,
+                "stonefurnace", 4000,
```

Comment:
You need to describe this change in the PR description. Actually, everything really.